index.Rmd

```diff
@@ -1,105 +1,299 @@
+I prepare the data by reading in the different bands that comprise the satelitte imagery. Each band refers to different spectrum.
 
-You can do bullets like this:
 
-* The first most important thing
-* The second most important thing
-* The third most important thing
+```{r, message=F, warning=F}
+Band1 <- ("B02.tif")
+Band2 <- ("B03.tif")
+Band3 <- ("B04.tif")
+Band4 <- ("B05.tif")
+Band5 <- ("B06.tif")
+Band6 <- ("B07.tif")
+Band7 <- ("B08.tif")
+
+band1 <- ("b01.tiff")
+band2 <- ("b02.tiff")
+band3 <- ("b03.tiff")
+band4 <- ("b04.tiff")
+band5 <- ("b05.tiff")
+band6 <- ("b06.tiff")
+band7 <- ("b07.tiff")
+band8 <- ("b08.tiff")
+band9 <- ("b09.tiff")
+```
 
-You can do numbers like this:
 
-1. The first most important thing
-2. The second most important thing
-3. The third most important thing
 
-See [http://rmarkdown.rstudio.com/](http://rmarkdown.rstudio.com/) for all the amazing things you can do.
+```{r, message=F, warning=F}
+library(raster)
+#This first part of the plot is an example on how it works with Landsat data for Earth
+Band1 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B02.tif")
+#(band1 <- raster("B02.tif"))
+Band2 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B03.tif")
+Band3 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B04.tif")
+Band4 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B05.tif")
+Band5 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B06.tif")
+Band6 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B07.tif")
+Band7 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B08.tif")
+
+
+#This first part  is showing how it works with Mars data
+band1 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b01.tiff")
```

Comment:
again - why upper and lower case files?  Are they different?

Reply:
Also, these files are on your desktop and will need to be stored in the repository or another public location.